includes/Dashboard.php

```diff
@@ -316,28 +330,36 @@ public function get_dashboard_settings() {
 		$settings = [
 			'id'         => 'web-stories-dashboard',
 			'config'     => [
-				'isRTL'              => is_rtl(),
-				'locale'             => ( new Locale() )->get_locale_settings(),
-				'newStoryURL'        => $new_story_url,
-				'editStoryURL'       => $edit_story_url,
-				'wpListURL'          => $classic_wp_list_url,
-				'assetsURL'          => trailingslashit( WEBSTORIES_ASSETS_URL ),
-				'cdnURL'             => trailingslashit( WEBSTORIES_CDN_URL ),
-				'version'            => WEBSTORIES_VERSION,
-				'encodeMarkup'       => $this->decoder->supports_decoding(),
-				'api'                => [
+				'isRTL'               => is_rtl(),
+				'locale'              => ( new Locale() )->get_locale_settings(),
+				'newStoryURL'         => $new_story_url,
+				'editStoryURL'        => $edit_story_url,
+				'wpListURL'           => $classic_wp_list_url,
+				'assetsURL'           => trailingslashit( WEBSTORIES_ASSETS_URL ),
+				'cdnURL'              => trailingslashit( WEBSTORIES_CDN_URL ),
+				'version'             => WEBSTORIES_VERSION,
+				'encodeMarkup'        => $this->decoder->supports_decoding(),
+				'api'                 => [
 					'stories'     => sprintf( '/web-stories/v1/%s', $rest_base ),
 					'media'       => '/web-stories/v1/media',
 					'currentUser' => '/wp/v2/users/me',
 					'users'       => '/wp/v2/users',
 					'templates'   => '/web-stories/v1/web-story-template',
 					'settings'    => '/wp/v2/settings',
 				],
-				'maxUpload'          => $max_upload_size,
-				'maxUploadFormatted' => size_format( $max_upload_size ),
-				'capabilities'       => [
+				'maxUpload'           => $max_upload_size,
+				'maxUploadFormatted'  => size_format( $max_upload_size ),
+				'capabilities'        => [
 					'canManageSettings' => current_user_can( 'manage_options' ),
 					'canUploadFiles'    => current_user_can( 'upload_files' ),
+					
+				],
+				'siteKitCapabilities' => [
+					'siteKitInstalled'      => $this->is_site_kit_plugin_installed(),
+					'siteKitActive'         => defined( 'GOOGLESITEKIT_VERSION' ),
+					'analyticsModuleActive' => false, // TODO: copy the logic from Analytics.php we need to share somehow.
+					'canInstallPlugins'     => current_user_can( 'install_plugins' ),
+					'canActivatePlugins'    => current_user_can( 'activate_plugin', 'google-site-kit/google-site-kit.php' ),
```

Comment:
What are these used for? Are they required? Why not put them as part of `capabilities` above?